packages/multichain-network-controller/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - The `AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS` now includes non-EVM testnets ([#5589](https://github.com/MetaMask/core/pull/5589))
+- Fix the condition to update the active network based on the `AccountsController:selectedAccountChange` event ([#5642](https://github.com/MetaMask/core/pull/5642))
```

Comment:
Does this fix a bug? If so, perhaps it should go under a "Fixed" header?

Reply:
good point

Reply:
done in https://github.com/MetaMask/core/pull/5642/commits/c051fbee7e515f5107bd5316f29b89b65a46e2e4